pkgs/data/themes/materia-theme-transparent/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  fetchFromGitHub,
+  materia-theme,
+}:
+materia-theme.overrideAttrs (oldAttrs: rec {
+  pname = "materia-theme-transparent";
+  version = "0-unstable-2021-03-22";
+
+  src = fetchFromGitHub {
+    owner = "ckissane";
+    repo = pname;
```

Comment:
```suggestion
materia-theme.overrideAttrs (oldAttrs: {
  pname = "materia-theme-transparent";
  version = "0-unstable-2021-03-22";

  src = fetchFromGitHub {
    owner = "ckissane";
    repo = "materia-theme-transparent";
```

Reply:
I'll get rid of the rec, but what is the point of hardcoding the repo? Is it to better help overrides? They would be overriding the src manually anyway. Thanks for the suggestions :)

Reply:
It is not recommended to overuse pname #277994

Reply:
Alright, done